### PR TITLE
Implement createSerializedExecutionRequest function

### PR DESCRIPTION
**Pull request by supl**
This PR implements createSerializedExecutionRequest function to serialize the contract execution request.

It has a different interface from the createSerializedContractExecutionRequest function.

### CODE REVIEW

**Review by jnmt — COMMENTED** [review]
Overall looking good, thank you! Just a minor point. Is it better to have a deprecated mark in the comment area of `createSerializedContractExecutionRequest()`?

File: index.js [hunk 1]
```diff
@@ -608,6 +608,48 @@ class ClientServiceBase {
...
+   * @param {string} [functionId=null]
+   * @param {Object|string} [functionArgument=null]
+   * @param {string} [nonce=null]
+   * @return {Promise<Uint8Array>}
+   * @throws {ClientError|Error}
+   */
+  async createSerializedExecutionRequest(
+      contractId,
+      contractArgument,
+      functionId,
```
Comment (jnmt) [inline index.js:624]:
```suggestion
      functionId = null,
```

**Comment by supl** [comment]
> Overall looking good, thank you! Just a minor point. Is it better to have a deprecated mark in the comment area of `createSerializedContractExecutionRequest()`?

Ah, Yes!
Thank you for the reminder!
Let me add it!

**Review by jnmt — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**